h2o-core/src/main/java/water/rapids/ast/prims/mungers/AstGroup.java

```diff
@@ -257,6 +267,14 @@ public double postPass(double ds[], long n) {
     }
   }
 
+  /**
+   * Allowed column types by which data can be grouped.
+   */
+  public enum GroupByColType {
```

Comment:
internally se use byte for column types see Vec.T_STR

Reply:
This makes sense. I changed it throughout the code to use the type constants from `Vec`.